src/vehicle_use.cpp

```diff
@@ -2041,14 +2167,20 @@ void vehicle::interact_with( const tripoint &pos, int interact_part )
             return;
         }
         case FILL_CONTAINER: {
-            g->u.siphon( *this, "water_clean" );
+            item_count_tuple item_tuple = select_comestible_liquid( comestible_liquids,
+                                          _( "Select a liquid" ) );
+            const item *comestible_ptr = std::get<0>( item_tuple );
+            if( comestible_ptr ) {
+                use_fill_container( std::get<0>( item_tuple ), ( has_chemlab || has_kitchen ), ( has_fridge ||
```

Comment:
```suggestion
                use_fill_container( std::get<0>( item_tuple ), has_chemlab || has_kitchen, has_fridge || has_freezer );
```
Brackets are redundant here.